inyoka_theme_default/templates/planet/index.html

```diff
@@ -77,15 +82,24 @@
                     <div class="text">{{ article.text }}</div>
                   </div>
                 </div>
-                <div class="planet-article-footer">
-                  <a href="{{ article|url|e }}">{% trans %}Permanent link{% endtrans %}</a>
+
+                <ul class="planet-article-footer">
```

Comment:
2. if i rely on 1. (as a bonus consequence of 3.) i removed the wrapper-div and it is one element less. Or should i remove the `</li>`-tags to sparse even more markup? :P